app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,22 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: int
+    ) -> None:
+        self._skill_level = skill_level
+        super().__init__(
+            nickname=nickname,
+            favourite_dish=favourite_dish
+        )
+
+    def player_info(self) -> str:
+        return f"Dwarf blacksmith {self.nickname} " \
+               f"with skill of the {self._skill_level} level"
```

Comment:
We don't recommend to use `/`, do the same without it. Fix in all other similar cases